io.itch.domestique_baston.java_jolt.yml

```diff
@@ -0,0 +1,28 @@
+id: io.itch.domestique_baston.java_jolt
+runtime: org.freedesktop.Platform
+runtime-version: '24.08'
+sdk: org.freedesktop.Sdk
+command: java_jolt
+finish-args:
+  - --device=dri
+  - --socket=x11
+  - --share=ipc
+  - --socket=pulseaudio
+modules:
+  - name: java_jolt
+    buildsystem: simple
+    build-commands:
+      - install -D 'JAVA JOLT!.x86_64' /app/bin/java_jolt
+      - install -D -m 0644 io.itch.domestique_baston.java_jolt.metainfo.xml /app/share/metainfo/$FLATPAK_ID.metainfo.xml
+      - install -D -m 0644 io.itch.domestique_baston.java_jolt.desktop /app/share/applications/$FLATPAK_ID.desktop
+      - install -D -m 0644 JJ.png /app/share/icons/hicolor/256x256/apps/$FLATPAK_ID.png
+    sources:
+    - type: archive
+      url: 'http://www.ferdiferdi.com/jj/JJ_LIN.zip'
```

Comment:
do you own this domain? It might be better to use it in the app id that `io.itch`, website does not support verification with itch.

Reply:
if you do, please link the game somewhere in the website.

Reply:
My colleague does own this domain, but he does not want the game to be directly associated with him or his website. So the game is hosted by itch.io under the label Domestique Baston, which is actually composed of a few people...

Reply:
We use the same sort of app ID for our first game (Domestique Baston) and intend to do so for a third, which has not yet been flatpakked.

Reply:
OK, that's fine, you will need to wait until the website supports itch login.